zrml/prediction-markets/src/lib.rs

```diff
@@ -1729,8 +1784,8 @@ mod pallet {
         /// which open at the same time as the specified market,
         /// and `m` is the number of market ids,
         /// which close at the same time as the specified market.
-        #[pallet::call_index(21)]
         #[pallet::weight(T::WeightInfo::close_trusted_market(CacheSize::get(), CacheSize::get()))]
+        #[pallet::call_index(22)]
         #[transactional]
         pub fn close_trusted_market(
```

Comment:
I assume this won't work, because close_trusted_market is already deployed, right?

Reply:
nevermind ;-) https://github.com/zeitgeistpm/zeitgeist/pull/1160/commits/425edd2dede1029d26228c4dc79329c5de24d240